src/components/cdTab.js

```diff
@@ -1,35 +1,187 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import Grid from '@mui/material/Grid'
-import { Skeleton } from '@mui/material'
+import {
+  Alert,
+  Box,
+  FormControlLabel,
+  FormGroup,
+  Switch,
+  Tooltip,
+} from '@mui/material'
 
 import MultilineTextViewer from './util/MultilineTextViewer'
+import { useFetchComponentDescriptor } from '../api/useFetch'
+import CenteredSpinner from './util/CenteredSpinner'
+import { SearchParamContext } from '../App'
 import { toYamlString } from '../util'
 
 
+const FetchAndView = ({
+  componentName,
+  componentVersion,
+  ocmRepo,
+  versionFilter,
+  setRaw,
+}) => {
+  // eslint-disable-next-line no-unused-vars
+  const [componentDescriptor, isLoading, isError, _] = useFetchComponentDescriptor({
```

Comment:
nit: let us be consistent here, i.e. either using `_` as dummy variable for all unused variables or disabling linter warning and not using `_`